assets/metadata.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import os
+import requests
```

Comment:
Why are you importing requests and dpath here? They're not being used.

Reply:
ah yes think this was left over from code we reused, will remove